contextual_lenses/contextual_lenses.py

```diff
@@ -20,129 +19,143 @@
 
 
 def max_pool(x, padding_mask=None, pad_constant=1e8):
-  """Apply padding, take maximum over sequence length axis."""
+    """Apply padding, take maximum over sequence length axis."""
+
+    if padding_mask is not None:
+        x = x * padding_mask
+        neg_mask = -pad_constant * (1 - padding_mask)
+        x = x + neg_mask
 
-  if padding_mask is not None:
-    x = x * padding_mask
-    neg_mask = - pad_constant * (1 - padding_mask)
-    x = x + neg_mask
-   
-  rep = jnp.max(x, axis=-2)
-   
-  return rep
+    rep = jnp.max(x, axis=-2)
+
+    return rep
 
 
 def mean_pool(x, padding_mask=None):
-  """Apply padding, take mean over sequence length axis."""
+    """Apply padding, take mean over sequence length axis."""
 
-  if padding_mask is not None:
-    x = x * padding_mask
-    rep = jnp.sum(x, axis=-2) / jnp.sum(padding_mask, axis=-2)
-  else:
-    rep = jnp.mean(x, axis=-2)
+    if padding_mask is not None:
+        x = x * padding_mask
+        rep = jnp.sum(x, axis=-2) / jnp.sum(padding_mask, axis=-2)
+    else:
+        rep = jnp.mean(x, axis=-2)
 
-  return rep
+    return rep
 
 
 def linear_max_pool(x, rep_size, padding_mask=None):
-  """Apply linear transformation + ReLU, apply padding,
+    """Apply linear transformation + ReLU, apply padding,
      take maximum over sequence length.
   """
-  
-  x = nn.Dense(
-        x,
-        rep_size,
-        kernel_init=nn.initializers.xavier_uniform(),
-        bias_init=nn.initializers.normal(stddev=1e-6))
-  
-  x = nn.relu(x)
-  
-  rep = max_pool(x, padding_mask=padding_mask)
-  
-  return rep
+
+    x = nn.Dense(x,
+                 rep_size,
+                 kernel_init=nn.initializers.xavier_uniform(),
+                 bias_init=nn.initializers.normal(stddev=1e-6))
+
+    x = nn.relu(x)
+
+    rep = max_pool(x, padding_mask=padding_mask)
+
+    return rep
 
 
 def linear_mean_pool(x, rep_size, padding_mask=None):
-  """Apply linear transformation + ReLU, apply padding,
+    """Apply linear transformation + ReLU, apply padding,
      take mean over sequence length.
   """
-    
-  x = nn.Dense(
-        x,
-        rep_size,
-        kernel_init=nn.initializers.xavier_uniform(),
-        bias_init=nn.initializers.normal(stddev=1e-6))
-  
-  x = nn.relu(x)
-  
-  rep = mean_pool(x, padding_mask=padding_mask)
-  
-  return rep
+
+    x = nn.Dense(x,
+                 rep_size,
+                 kernel_init=nn.initializers.xavier_uniform(),
+                 bias_init=nn.initializers.normal(stddev=1e-6))
+
+    x = nn.relu(x)
+
+    rep = mean_pool(x, padding_mask=padding_mask)
+
+    return rep
 
 
 class GatedConv(nn.Module):
-  """Gated Convolutional lens followed by max pooling,
+    """Gated Convolutional lens followed by max pooling,
```

Comment:
Did you run a formatter over it? If so, curious which one. Looks like a lot of lines are changed with formatting diffs

Think a good choice is either `black` or yapf with the `google` style guide: https://github.com/google/yapf: `yapf --style google *.py`

Also reasonable to do in a followup PR that just does formatting & no content

Reply:
yep ran yapf -ir . on the scripts (not the ones in google-research repo)